app/policies/wiki_policy.rb

```diff
@@ -0,0 +1,7 @@
+class WikiPolicy < ApplicationPolicy
+
+  def index?
+    true
```

Comment:
ah, you're overriding the ApplicationPolicy#index policy.  Ok, so users will be able to see the wiki index.  But what about destroy?